tests/unit/test_speechShortcutKeys.py

```diff
@@ -12,11 +12,22 @@
 	_getKeyboardShortcutSpeech,
 	getKeyboardShortcutsSpeech,
 )
+import speech.shortcutKeys
 from speech.commands import CharacterModeCommand
 
 
+original_shouldUseSpellingFunctionality = speech.shortcutKeys.shouldUseSpellingFunctionality
+
 class Test_getKeyboardShortcutSpeech(unittest.TestCase):
 
+	@classmethod
+	def setUpClass(cls):
```

Comment:
Have you considered using `unittest.mock.patch` for replacing the function during the tests? When used as a decorator on these test classes you'll have much less of boilerplate code.